fee_allocator/constants.py

```diff
@@ -0,0 +1,16 @@
+FEE_CONSTANTS_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/protocol_fees_constants.json"
+CORE_POOLS_URL = "https://raw.githubusercontent.com/BalancerMaxis/bal_addresses/main/outputs/core_pools.json"
+REROUTE_CONFIG_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/core_pools_rerouting.json"
+POOL_OVERRIDES_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/ae9cfa3d627dfb66d1fba6f824c316e4769bbf5a/config/pool_incentives_overrides.json"
+SNAPSHOT_URL = "https://hub.snapshot.org/graphql?"
+HH_API_URL = "https://api.hiddenhand.finance/proposal"
+GAUGE_MAPPING_URL = "https://raw.githubusercontent.com/aurafinance/aura-contracts/main/tasks/snapshot/gauge_choices.json"
+
+EXPLORER_URLS = {
```

Comment:
Can we make a class in bal_tools for explorer fetching?  I see this everywhere.

Reply:
this is actually already added in bal_addresses: https://github.com/BalancerMaxis/bal_addresses/blob/8fd72eacd69081a407a36192f8fbf60fd9192155/extras/chains.json#L32 so we can just reference from there